editor.planx.uk/src/@planx/components/MapAndLabel/Public.tsx

```diff
@@ -1,13 +1,35 @@
-import React from "react";
+import { alpha } from "@mui/material/styles";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React, { useState } from "react";
 
 import Card from "../shared/Preview/Card";
 import CardHeader from "../shared/Preview/CardHeader";
+import { MapContainer } from "../shared/Preview/MapContainer";
 import { PublicProps } from "../ui";
 import { MapAndLabel } from "./model";
 
 type Props = PublicProps<MapAndLabel>;
 
 function MapAndLabelComponent(props: Props) {
+  const teamSettings = useStore.getState().teamSettings;
+
+  const defaultBboxJSON = {
+    type: "Feature",
+    geometry: {
+      type: "Polygon",
+      coordinates: [
+        [
+          [1.9134116, 49.528423],
+          [1.9134116, 61.331151],
+          [1.9134116, 61.331151],
+          [-10.76418, 61.331151],
+          [-10.76418, 49.528423],
+        ],
+      ],
+    },
+    properties: {},
+  };
```

Comment:
nit: it'd be helpful to have a code comment that explains what this extent is and where you got it (eg full UK? or only England?)

We can also just pass `undefined` to the `clipGeojsonData` prop & the map repo will handle it accordingly ! https://github.com/theopensystemslab/map/blob/main/src/components/my-map/index.ts#L213-L218

Reply:
Removed this now